core/src/components/datetime/datetime.tsx

```diff
@@ -1859,6 +1859,9 @@ export class Datetime implements ComponentInterface {
     const prevMonthDisabled = isPrevMonthDisabled(this.workingParts, this.minParts, this.maxParts);
     const nextMonthDisabled = isNextMonthDisabled(this.workingParts, this.maxParts);
 
+    // don't use the inheritAttributes util because it removes dir from the host, and we still need that
+    const hostDir = this.el.getAttribute('dir') || undefined;
```

Comment:
Should we be using the `isRTL` utility?

Reply:
Checking the `dir` on the `document` is covered by the icons themselves, which do go through the util, so using it again here would be redundant. I could add it in for safety though if you'd rather?

Reply:
I'm fine either way, mainly was curious 👍